### PR TITLE
Fixes #1794 - Prevent submit by enter in bugform

### LINKED ISSUE #1794: Enter in description textbox submits form
As described in this bug: https://github.com/webcompat/web-bugs/issues/9706

If you press enter in the description textbox, it submits the form before you've (potentially) finished filling it out.

Maybe we should prevent this behaviour and force the user to click the button?

### PR DESCRIPTION
**Pull request by Regaddi**
r? @miketaylr @zoepage

### CODE REVIEW

**Comment by zoepage** [comment]
r+

File: webcompat/static/js/lib/bugform.js [hunk 1]
```diff
@@ -94,6 +94,17 @@ function BugForm() {
     );
     this.form.on("submit", _.bind(this.maybeUploadImage, this));
 
+    // prevent submit by hitting enter key for single line input fields
+    this.urlField.on("keypress", _.bind(this.preventSubmitByEnter, this));
```
Comment (miketaylr) [inline webcompat/static/js/lib/bugform.js]: Maybe we could make this a little bit more DRY via event delegation, i.e., adding the listener on the form element itself, and checking that its target is an `input` (and not `textarea`) inside `preventSubmitByEnter`?

**Comment by Regaddi** [comment]
@miketaylr I reduced to only one event listener.

File: webcompat/static/js/lib/bugform.js [hunk 2]
```diff
@@ -94,6 +94,13 @@ function BugForm() {
     );
     this.form.on("submit", _.bind(this.maybeUploadImage, this));
 
+    // prevent submit by hitting enter key for single line input fields
+    this.form.on(
+      "keypress",
+      ":input:not(textarea)",
```
Comment (miketaylr) [inline webcompat/static/js/lib/bugform.js]: clever, i like it!

**Pull request merged**